signature/policy_types.go

```diff
@@ -111,11 +111,14 @@ type prSignedBaseLayer struct {
 type prSigstoreSigned struct {
 	prCommon
 
-	// KeyPath is a pathname to a local file containing the trusted key. Exactly one of KeyPath, KeyData, Fulcio must be specified.
+	// KeyPath is a pathname to a local file containing the trusted key. Exactly one of KeyPath, KeyPaths, KeyData, KeyDatas or Fulcio must be specified.
```

Comment:
In the code above it says "and fulcio" and here it says "or".

Reply:
I’m not sure there is a difference in meaning, but being consistent is nice. Fixed.